easybuild/easyconfigs/t/treelib/treelib-1.8.0-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'PythonPackage'
+
+name = 'treelib'
```

Comment:
seems a bit overkill to have a separate easyconfig for such a small package.
maybe include it into nvidia-cutlass?

Reply:
I wasn't sure if it isn't useful for other easyconfigs or on its own given it is a very generic package.

With PythonBundles we tend to introduce different versions of PythonPackages without detecting it on CI.   

But [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) applies so can do